dist/rei-dot-com/js/foundations/cdr-color-background.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  "CdrColorBackgroundAccordionHover": "#fafbf9",
```

Comment:
I get why these are here but I wonder if there would be a way to remove the component tokens from these foundations -- really what I would expect here would be something more like 

  "CdrColorBackgroundPrimary": "#ffffff",
  "CdrColorBackgroundSecondary": "#F4F2ED",
  "CdrColorBackgroundSale": "#c7370f",
  "CdrColorBackgroundBrandSpruce": "#1f513f",
  "CdrColorBackgroundSuccess": "#ecf9e6",
  "CdrColorBackgroundInfo": "#e2f4fe",
  "CdrColorBackgroundWarning": "#fdf6e2",
  "CdrColorBackgroundError": "#fcefe4",